### PR TITLE
Add invoice picking address 

### PR DESCRIPTION
**Pull request by eLBati**
Adds partner delivery address on invoice report.
Delivery address is retrieved from sale order linked to invoice, or from delivery order, when invoicing is based on delivery.

### CODE REVIEW

File: account_invoice_picking_address/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': "Invoice Report Picking Address",
+    'summary': 'Add Picking Address on invoice template',
+    'version': '1.0',
```
Comment (tafaRU) [inline account_invoice_picking_address/__openerp__.py]: Please update version according to https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

File: account_invoice_picking_address/account.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    delivery_address_id = fields.Many2one(
+        comodel_name='res.partner',
+        string='Delivery Address',
+        compute='_compute_delivery_address')
+    sale_order_ids = fields.Many2many(
+        'sale.order', 'sale_order_invoice_rel', 'invoice_id', 'order_id',
```
Comment (tafaRU) [inline account_invoice_picking_address/account.py:32]: Just attribute comodel_name is mandatory so you could remove the others as optional.

**Comment by archetipo** [comment]
:+1:

**Comment by gurneyalex** [comment]
:+1:

**Pull request merged**